packages/prompts/src/password.ts

```diff
@@ -6,6 +6,7 @@ export interface PasswordOptions extends CommonOptions {
 	message: string;
 	mask?: string;
 	validate?: (value: string | undefined) => string | Error | undefined;
+	reset?: boolean;
```

Comment:
i wonder if we should be clearer about this - naming it something like `resetOnError` or `clearOnError`